### PR TITLE
[LIB-526] As a user I have documentation for pinger.js

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: tutorials/04.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+This tutorial will show you how to monitor the connection to the platform.
```
Comment (dancio) [inline tutorials/04.md]: just `04.md`?

Comment (jakubbilko) [inline tutorials/04.md]: Hm, some kind of mistake

**Comment by dancio** [comment]
if u want to skip CI build just add `[ci skip]` to commit message, lgtm

**Comment by jakubbilko** [comment]
Roger!

**Pull request merged**